examples/onnxruntime/training/language-modeling/run_clm.py

```diff
@@ -506,7 +506,10 @@ def tokenize_function(examples):
                 f"The tokenizer picked seems to have a very large `model_max_length` ({tokenizer.model_max_length}). "
                 f"Using block_size={min(1024, max_pos_embeddings)} instead. You can change that default value by passing --block_size xxx."
             )
+        if max_pos_embeddings > 0:
```

Comment:
What about checking if `max_pos_embeddings>0` while fetching its value:
https://github.com/huggingface/optimum/blob/6c80244465cc92fd46c0381a12a287ddc8456e3f/examples/onnxruntime/training/language-modeling/run_clm.py#L496-L497
something like
```python
max_pos_embeddings = getattr(config, "max_position_embeddings", 1024)
if max_pos_embeddings < 0:
    max_pos_embeddings = 1024

```

Reply:
That also works, I changed it